src/components/WorkerPool.tsx

```
@@ -50,58 +50,55 @@ export const WorkerPool: React.FC<WorkerPoolProps> = ({
     <div className="worker-pool">
       <div className="worker-pool-header">
         <h3>Workers</h3>
-        <button 
+        <button
           className="add-worker-button"
           onClick={handleAddClick}
           title="Add a new worker"
         >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)